app/api/domains/route.ts

```diff
@@ -64,25 +57,31 @@ setInterval(() => {
   currentDomainNameAvailabilityCheck = checkDomainNamesAvailability();
 }, 1000);
 
-export async function POST(request: Request) {
+export async function POST(request: NextRequest) {
   let { description } = await request.json();
 
   // Make sure description is 100 characters or less
   description = description.slice(0, 100);
 
-  const completion = await openai.createChatCompletion({
-    model: "gpt-3.5-turbo",
-    messages: [
-      {
-        role: "user",
-        content: `List some suitable domain names for my project in CSV format. Description of my project: "${description}"`,
-      },
-    ],
-  });
+  const completion = await fetch("https://api.openai.com/v1/chat/completions", {
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${process.env.OPENAI_API_KEY}`,
+    },
+    body: JSON.stringify({
+      model: "gpt-3.5-turbo",
+      messages: [
+        {
+          role: "user",
+          content: `List some suitable domain names for my project in CSV format. Description of my project: "${description}"`,
+        },
+      ],
+    }),
+  }).then((response) => response.json());
```

Comment:
Switched from `openai` package to `fetch` so we can be sure that we're not importing anything that's not supported by Edge Runtime.